README.md

```diff
@@ -36,6 +36,19 @@ For more information about features go to [our Swift documentation](https://meil
 
 ## Get started
 
+### Cocoapods
+
+[CocoaPods](https://cocoapods.org/) is a dependency manager for Cocoa projects.
+
+**MeiliSearch-Swift** is available through [CocoaPods](https://cocoapods.org). To install
+it, simply add the following line to your Podfile:
+
+```ruby
```

Comment:
Cocoapods is a ruby command, not Swift. If you check the auto generated README from Cocoapods it will result into this:

<img width="286" alt="Screenshot 2021-05-03 at 16 20 39" src="https://user-images.githubusercontent.com/5521682/116895918-93500c00-ac2b-11eb-9893-0f5760bc5177.png">

From: https://raw.githubusercontent.com/Alamofire/Alamofire/master/README.md

Reply:
My bad I went to fast on this one :)